nats-base-client/service.ts

```diff
@@ -551,19 +579,14 @@ export class ServiceImpl implements Service {
         this.close(err);
         return Promise.reject(err);
       }
-
-      msg?.respond(jc.encode(this.config.schema));
+      msg?.respond(this.schema);
       return Promise.resolve();
     };
 
     this.addInternalHandler(ServiceVerb.PING, pingHandler);
     this.addInternalHandler(ServiceVerb.STATS, statsHandler);
     this.addInternalHandler(ServiceVerb.INFO, infoHandler);
-    if (
-      this.config.schema?.request !== "" || this.config.schema?.response !== ""
-    ) {
-      this.addInternalHandler(ServiceVerb.SCHEMA, schemaHandler);
-    }
+    this.addInternalHandler(ServiceVerb.SCHEMA, schemaHandler);
```

Comment:
Do we want to always register this handler? I though we only want it if schema was set on the service config.

Reply:
I was thinking yes, because then you can see it and understand they didn't configure it.